tools/m2ctx.py

```diff
@@ -25,6 +25,7 @@
     "-D__asm__(test...)=",
     "-ffreestanding",
     "-DM2CTX",
+    "-DVERSION_PAL",
```

Comment:
Revert this probably

Reply:
It's intentional. Without it, m2ctx is useless currently, since all of US is finished. Parameterizing it would be nice, but this is a quick fix in the meantime